automation/taskcluster/lib/tasks.py

```diff
@@ -118,14 +118,15 @@ def craft_ui_tests_task(self):
         )
 
         command = '{} && {}'.format(checkout_command, test_command)
+        #    "image": "mozillamobile/android-components:1.17",
         payload = {
             "env": { 
                 "GOOGLE_PROJECT": "moz-android-components-230120",
                 "GOOGLE_APPLICATION_CREDENTIALS": ".firebase_token.json",
             },
             "features": features,
             "maxRunTime": 7200,
-            "image": "mozillamobile/android-components:1.17",
+            "image": "rpappalax/a-c-demo2",
```

Comment:
setting this temporarily to my working docker image (with required updates) to demonstrate that tests pass, since mozilla-mobile/android-components image will need to be updated manually